typegraph/core/src/runtimes/grpc/mod.rs

```diff
@@ -46,3 +39,30 @@ impl MaterializerConverter for GrpcMaterializer {
         })
     }
 }
+
+fn get_gprc_data(runtime_id: RuntimeId) -> Rc<GrpcRuntimeData> {
+    match Store::get_runtime(runtime_id).unwrap() {
+        Runtime::Grpc(data) => data,
+        _ => unreachable!(),
+    }
+}
```

Comment:
Minimizing unwrap would be best imo, and instead always return Results (especially if assertions can break like in this case), wasm panics are pretty much unreadable at the time being.